### ISSUE TITLE
Accessibility resources

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/webvr-content-workshop#4]
I've been made aware of at least two resources that can be useful for thinking of A11Y and VR:
* https://www.w3.org/WAI/APA/task-forces/research-questions/wiki/Accessibility_of_Virtual_Reality developed by the WAI APA Research task force
* https://www.gamasutra.com/blogs/IanHamilton/20161031/284491/VR__accessibility.php (submitted to the workshop mailing list)

We should build on those for preparing the accessibility session at the workshop

**Comment by Utopiah** [comment]
Roland Dubois (@rdub80) does a lot of work building [GraVR.io](http://www.gravr.io/) and did a recent presentation at NYVR on [Approaches to Accessibility in WebVR](https://photo.a2zinc.net/PhotoPlus2017/public/SessionDetails.aspx?FromPage=Sessions.aspx&SessionID=2642&SessionDateID=104). He also did a great A-Frame demo https://github.com/rdub80/gravr-a11y-demo that should become soon a component.

**Comment by dontcallmedom** [comment]
do you know if anyone could serve as a conduit to these findings? This sounds like a very interesting source of information

**Comment by Utopiah** [comment]
[Roland is aware of this issue](https://twitter.com/utopiah/status/928523603351691264) but if he can not attend we'll try to find a conduit.

**Comment by rdub80** [comment]
@dontcallmedom I submitted the application to attend, I would be happy to join the working group in  Brussels, @Utopiah thanks for the intro

**Comment by dontcallmedom** [comment]
@rdub80 looking forward to meeting you there!

**Comment by delapuente** [comment]
Another interesting resource:
http://www.ablegamers.org/thoughts-on-accessibility-and-vr/

**Comment by rdub80** [comment]
Even though the ILMxLAB Survey is very much focussed around 6dof / room
scale, it is an interesting read: https://goo.gl/3fDn3N

**Comment by CarinC** [comment]
Hi all, 

I was sadly unable to attend this event, or to post anything on time for it, but would like to share some info that might be useful.

---

**Projections** might help users who can't wear VR headsets, e.g. by:
1. **BroomX** - 360/VR projector (interact with the projection via a mobile device): http://www.broomx.com/ 
2. **SCALee** - portable VR projector: https://www.fitness-gaming.com/news/home-fitness/scalee-vr-technology-lets-users-become-part-of-virtual-worlds.html  
3. **LUMOplay** - affordable 2D projector (can it be linked/hacked to work with other content than their own?): https://www.lumoplay.com/ and https://www.fitness-gaming.com/news/home-fitness/lumo-play-turns-childrens-rooms-into-interactive-playgrounds.html
4. **Domes** with projectors inside, for events or permanent fixtures (to be hired/bought by companies rather than consumers), e.g. from Igloo: http://www.igloovision.com/ or Odome (also under water): https://www.odome.co.uk/

---

**Haptic feedback** might help users navigate and find/manipulate assets in VR, e.g. provided by:
1. **VR Touch** - affordable small wireless finger add-on for haptic feedback with accurate dynamic pressure: https://www.gotouchvr.com/
2. **HaptX Gloves** - expensive, bulky & wired, but powerful haptic gloves: https://haptx.com/ and https://www.engadget.com/2017/11/20/haptx-gloves-vr/
There are many other glove providers, ... 
3. **Ultrahaptics** - panels that create interactive haptic objects in the air by ultrasound, but are still in early stages: https://www.ultrahaptics.com/

---

**Hand gestures** might help users interact in a more natural way than with hand-held controllers (with learned interactions), e.g. through:
1. **Leapmotion** - small and cheap $80 gesture-recognition camera: https://www.leapmotion.com
2. **Hi5 VR GLOVE** - wireless consumer VR glove (only for HTC Vive & Noitom’s Project Alice - push them to expand this! :-) ): https://hi5vrglove.com/
3. **HaptX Gloves** - wired, bulky & expensive, but powerful haptic gloves (probably also only for certain platforms): https://haptx.com/ and https://www.engadget.com/2017/11/20/haptx-gloves-vr/

---

**Computer Vision & AI** might help describe the views in VR environments (either in spoken text or subtitles), now or in the near future, as illustrated by e.g. this apps, and these 2 APIs:
1. **Microsoft's free 'Seeing AI' app** (still using an image, not video stream, at the moment) 
- Info: https://www.microsoft.com/en-us/seeing-ai/
- Video of demo: https://www.youtube.com/watch?v=4krZgdX_WT8&feature=player_embedded 
2. **Microsoft's Computer Vision API** (try the API on your own image!): https://azure.microsoft.com/en-gb/services/cognitive-services/computer-vision/
3. **Google Video Intelligence API** (ony gives words, not phrases, at the moment?): https://cloud.google.com/video-intelligence/

---

The examples listed above are of course just that: a few easily found 'examples'. You can find many other examples for each of these topics, and one might fit some specific need just perfectly.

**Please therefore share your own examples here too**.

Many thanks, 

CarinC
